app/src/main/kotlin/com/wire/android/mapper/MessageContentMapper.kt

```diff
@@ -24,7 +24,7 @@ import com.wire.kalium.logic.feature.asset.GetMessageAssetUseCase
 import com.wire.kalium.logic.feature.asset.MessageAssetResult
 import com.wire.kalium.logic.util.isGreaterThan
 import javax.inject.Inject
-import com.wire.android.ui.home.conversations.model.MessageContent as UIMessageContent
+import com.wire.android.ui.home.conversations.model.UIMessageContent as UIMessageContent
```

Comment:
Should we drop now this import cast?

Reply:
Oooh yeah. Forgot to remove it after the rename :) Fixed